packages/flame/test/util/mock_gesture_events.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/gestures.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/extensions.dart';
+
+TapDownInfo createTapDownEvent({
```

Comment:
can be a followup, but would be good to have a test that actually tests the coordinate transforms with a complex camera + viewport setup

Reply:
We can do that when we fix the viewport